### DISCUSSION TITLE
How would you like to be onboarded?

### DISCUSSION THREAD

**Discussion by arkRedM** [athensresearch/athens#1228] [General]
https://github.com/athensresearch/athens/issues/753

Relevant comments: 
https://github.com/athensresearch/athens/issues/753#issuecomment-846008113 https://github.com/athensresearch/athens/issues/753#issuecomment-846467267

I was wondering how you got onboarded and what was the "hook" that made you stick around for Athens?
Was there some youtube video explaining the concept or a friend explaining the use case?

Basically, we want to bring that same experience to anyone who comes to Athens for the first time. https://github.com/athensresearch/athens/issues/753#issuecomment-846008113 This gives a basic idea of how it can be done and https://github.com/athensresearch/athens/issues/753#issuecomment-846467267 jeff's suggestion to make user do an action(write and see the links) is pretty useful.

How do you folks think we can extend/structure this? Please do share the youtube videos or the resources that "hooked" you to Athens or bi-directional thinking in general

Thanks!

**Comment by jefftangx** [comment]
https://saga.so/ has a solid onboarding

**Reply by arkRedM** [reply]
I loved it. Will get back with what we best for us and let's iterate on that.

**Reply by arkRedM** [reply]
ideas 

- a step tracker is neat.
- Keeping the core section of the app open is good(in our case It will be a page and all others hidden)
   -  spotlight seems less effective now
   -   Revealing sections in each step while explaining the use case
- Actions and being interactive is more valuable("hook")
   - Making it personalized with a name and page is very nice 
- I didn't like the gif, more distracting and less useful.

**Comment by arkRedM** [comment]
@tangjeff0  steps 
1. On first open - show a modal and ask for a name and big CTA to continue
    - default name is Socrates
2. Next screen is todays' daily note and like saga a sidebar
   - no toolbar or other sections visible, except page and onboarding sidebar
3. Showcase linking by asking the user to type [[]]
   - probably ask the user to go to type their own name
4. Ask to navigate to page
5. Showcase linked and unlinked
6. Unhide/show app toolbar
   - Ask if roam merge is needed. 
   - Make user Click and navigate to all pages and show message 
   - Make user Click and navigate to graph and show message
7. End message with modal and on close show welcome page (it will take care of rest of onboarding. ideally we should also include a help popup or shortcuts sections)

Extra
1. Skip button everywhere(except first maybe) 
2. Pick you where you left off if app closed or reloaded
3. List of steps and progress bar as each step is completed. 
4. highlight a section or bring attention with a blinking dot (or will look for alternatives)

It's close to saga's onboarding with our own feature sets

**Reply by arkRedM** [reply]
@tangjeff0 thoughts?

**Reply by arkRedM** [reply]
@jsmorabito. I was going through discord and you probably are the one who answered the most user questions. What do you think is the best set of things here that will reduce your work and let users discover/use features? 
Also what is the most undiscovered feature that becomes extremely valuable once it's known and more often our users miss?

**Reply by jsmorabito** [reply]
although I've answered a lot of user questions I'd say very few if any were about basic functionality which could be due to a variety of factors. 

 - for step 3. maybe mention that its possible to make pages with # and #[[]] and that theyre functionally the same? (might be awkward to have so much text not sure...)
 - one possible extra feature thatd be worth highlighting is the cmd-k 'athena' search and/or the search bar, i think that would fit naturally after step 4. ie heres another way to navigate to pages and search for content
 - also for step 3 maybe the text example could be "[[Athena]] is the goddess of [[knowledge]]" so that they have a connection in the linked references?

This all looks like a very solid start! I can mock up the flow here in figma as well so we can iterate as needed https://www.figma.com/file/zpnDCLykhdQyq5FANJb3Vz/Athens-Design-Sandbox?node-id=141%3A106 no pressure or need to add to it you should just get started working! 💪

**Comment by MikeBee** [comment]
I think the above items (scripted walkthrough of basic features) is nice, but I think it would be beneficial to look at the whole ecosystem.  Most of the potential user base has not picked up the app yet, why?  My journey was reading a few articles on medium and the like, to understand what I was missing.  Then I started hearing about some wider known names (Roam and Notion).  I do quite a few youtube searches to see and understand the value, and how it would fit my personal needs.  So I would recommend covering the following:
(a) get more information articles and videos about Athens and what makes it different,
(b) continual articles on sites like Medium describing different uses
(c) have the basic scripted walkthrough as an entry step
(d) have a better "beginner" entry place for new users to ask questions -- for me, discord was a barrier and not an easy place to know where and what to ask
(e) develop more advanced videos, and even paid courses, to provide the deeper skills
(f) follow this up with wikis, sites, tutorials, discord help on the advanced and intermediate functions (I think ClojureFam is a good attempt at this)

**Reply by joelhans** [reply]
Thanks for the feedback, @MikeBee!

We've had a few discussions lately, here on GitHub and in the Sunday community calls, about how Discord can be a barrier to entry for those who haven't used the platform before. Of course, GitHub kind of suffers from the same problem. If you feel like someone else (Roam, Notion, etc) really nailed this low barrier to asking questions, I'd love to learn from it.

Content like you mention in a/b/e/f is definitely on the medium-term roadmap, although we still have a big [handbook project](https://github.com/athensresearch/handbook/projects/3) ahead of us along with optimizing what we're already running. If you're interested in helping with content, even brainstorming different use cases we could talk about, that would be super welcome. Let me know!
